macros/GroupData.json

```diff
@@ -430,10 +430,12 @@
                             "Window: gamepaddisconnected" ]
         },
         "Geolocation API": {
-            "interfaces": [ "Coordinates",
-                            "Geolocation",
-                            "Position",
-                            "PositionError",
+            "overview":   [ "Geolocation API" ],
+            "guides":     [ "/docs/Web/API/Geolocation_API/Using" ],
+            "interfaces": [ "Geolocation",
+                            "GeolocationCoordinates",
+                            "GeolocationPosition",
+                            "GeolocationPositionError",
                             "PositionOptions" ],
```

Comment:
Honestly, I forgot what we've decided and implemented, but there is a "dictionaries" property now and `PositionOptions` is actually a dictionary, not an interface.